tests/unit/src/main/scala/tests/markdown/BaseMarkdownSuite.scala

```diff
@@ -73,8 +73,9 @@ abstract class BaseMarkdownSuite extends tests.BaseSuite {
         fansi.Str(myStdout.toString).plainText.trimLineEnds,
         postProcessObtained
       )
+
       assertNoDiff(
-        Compat(obtainedErrors, compat, postProcessObtained),
+        Compat(obtainedErrors, Map.empty, postProcessObtained),
```

Comment:
I looked at this and couldn't quite understand - my thinking is that we don't want to apply compat layers to **obtained** output, otherwise it resolves to the expectation itself and the test trivially passes (which was the reason that before some of the tests for errors were not doing anything)